nixos/doc/manual/from_md/configuration/bootloader-external.section.xml

```diff
@@ -0,0 +1,41 @@
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="sec-bootloader-external">
```

Comment:
Is that file duplicated? I think so.

Reply:
What makes you think so?

Reply:
You're right, it's a dupe of https://github.com/NixOS/nixpkgs/pull/172237/files#diff-8639dc73210a20178e2d73b591a7c77402873cf0b32f58eed88d08493c7edea3, which should be automatically populated thanks to `meta.doc`. Fixed.